bitnami/mlflow/templates/_helpers.tpl

```diff
@@ -648,6 +654,33 @@ Return the S3 secret access key inside the secret
     {{- end -}}
 {{- end -}}
 
+{{/*
+Return whether GCS is enabled
+*/}}
+{{- define "mlflow.v0.gcs.enabled" -}}
+    {{- if and (not .Values.minio.enabled) (not .Values.externalS3.host) .Values.externalGCS.bucket -}}
+        {{- true }}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Return whether artifacts should be served from GCS
+*/}}
+{{- define "mlflow.v0.gcs.serveArtifacts" -}}
+    {{- if .Values.externalGCS.serveArtifacts  -}}
+        {{- true }}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Return the GCS bucket
+*/}}
+{{- define "mlflow.v0.gcs.bucket" -}}
+    {{- if and (not .Values.minio.enabled) (not .Values.externalS3.host) .Values.externalGCS.bucket -}}
+        {{- print .Values.externalGCS.bucket -}}
+    {{- end -}}
+{{- end -}}
```

Comment:
I think this helper is not adding any value. We could just get the value from `.Values.externalGCS.bucket` as we are already wrapping the calls to the helper with the same condition, right?

```suggestion
```

I guess it's to follow the same approach than for `mlflow.v0.s3.bucket`, but in that case, the helper has a relevant if/else.